src/main/java/liquibase/ext/opensearch/database/OpenSearchConnection.java

```diff
@@ -144,7 +145,7 @@ private void connect() {
                     try {
                         sslcontext = SSLContextBuilder
                                 .create()
-                                .loadTrustMaterial(null, (chains, authType) -> true)
+                                .loadTrustMaterial(null, new TrustAllStrategy())
```

Comment:
I think this is fine for tests but not production, the `TrustAllStrategy is unsafe

Reply:
this is unrelated to this PR (sorry, i just snuck it in here, but at least as an atomic commit) - this commit doesn't change the behaviour at all, it's just making it more obvious what's happening here. there's #29 to improve this situation (but no clear idea on how to solve it yet).